runtime/wasm/src/host.rs

```diff
@@ -94,7 +97,11 @@ impl RuntimeHost {
             link_resolver: link_resolver.clone(),
         };
 
-        info!(logger, "Loading WASM runtime"; "data_source" => &config.data_source.name);
+        let name = config.data_source.name.clone();
+        info!(logger, "Loading WASM runtime"; "data_source" => &name);
```

Comment:
Is the `clone()` above necessary? Can we not pass `&config.data_source.name` to `info!`?

Reply:
It's not clear but it's because we move it into the thread so we can also log the name on shutdown.

Reply:
Ah, good.